.circleci/config.yml

```diff
@@ -2,12 +2,12 @@ version: 2
 jobs:
   build:
     docker:
-      - image: cimg/node:14.16
+      - image: cimg/node:20.9.0
         environment:
-          HUGO_VERSION: "0.81.0"
-          S3DEPLOY_VERSION: "2.3.5"
+          HUGO_VERSION: "0.120.4"
```

Comment:
The `HUGO_VERSION` variable is not longer used in the deploy process and can probably be used. We moved from downloading static `hugo` binaries to loading Hugo as a node dependency and executing it with `npx`. Looks like this didn't get removed when that change was made.

```suggestion
```

Reply:
Line was removed, as requested.